### ISSUE TITLE
Bug: XAML Controls Gallery (Debug): Accessibility: [MAS 2.4.7 - Focus Visible]: The keyboard focus is not visible on the Pager menu (Carousals) of the "PipsPager Window"

### ISSUE THREAD

**Issue by v-amnat** [microsoft/microsoft-ui-xaml#4272]
**Tags**
Accessibility, A11yCT, A11yMAS, A11yMediumImpact, MAS2.4.7, Keyboard, Sev3, XAML Controls Gallery (Debug), Win32, Wipro

**Steps to reproduce the bug**
1. Launch the XAML Controls Gallery (Debug) App
2. Go to the Scrolling-> Pips Pager.
3. Hit on Tab key to traverse the "Pager menu (Carousals)"
4. Observe the keyboard focus on the "Pager menu (Carousals)"

**Actual behavior**
The keyboard focus focus traverses the carousals' pager menu, but its not visible on the Pager menu (Carousals) of the "PipsPager Window"
Note: Although Narrator focus is visible on the same.

**Expected behavior**
The keyboard focus is not visible on the Pager menu (Carousals) of the "PipsPager Window"

**User Impact**
If keyboard focus is not visible on the on carousals' pager menu, then it could be difficult for the Keyboard user to understand its availability. 

**Screenshots**
[KB-Scrolling-PipsPager.zip](https://github.com/microsoft/microsoft-ui-xaml/files/6019810/KB-Scrolling-PipsPager.zip)

**Version Info**
Microsoft OS: 21313.1000
App: XAML Controls Gallery (Debug)

**Comment by StephenLPeters** [comment]
@stmoy FYi
